### PR TITLE
new(blog): rules and helm chart 3.0.0

### PR DESCRIPTION
**Pull request by LucaGuerra**
Signed-off-by: Luca Guerra <luca@guerra.sh>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

**What this PR does / why we need it**:

Add a simple guide in our blog for the most common use cases of the new Helm chart

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/rules-helm-chart-3-0-0.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Rules basics for the Falco 3.0.0 Helm chart
```
Comment (jsalinas29) [inline content/en/blog/rules-helm-chart-3-0-0.md:2]:
```suggestion
title: Rule basics for Falco 3.0.0 Helm chart
```

File: content/en/blog/rules-helm-chart-3-0-0.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+date: 2023-02-09
+author: Luca Guerra
+slug: rules-helm-chart-3-0-0
+---
+
+# Rules basics for the Falco 3.0.0 Helm chart
+
+The new Falco Helm chart 3.0.0 ([full documentation](https://github.com/falcosecurity/charts/blob/master/falco/README.md), [upgrade information](https://github.com/falcosecurity/charts/blob/master/falco/BREAKING-CHANGES.md#300)) comes with a new way to automatically update the Falco rules that are currently loaded. Of course, you can enable, disable and configure this functionality to your liking. Below, we list a number of common basic use cases and how to easily configure Falco for those:
```
Comment (jsalinas29) [inline content/en/blog/rules-helm-chart-3-0-0.md:10]:
```suggestion
The new Falco Helm chart 3.0.0 ([full documentation](https://github.com/falcosecurity/charts/blob/master/falco/README.md), [upgrade information](https://github.com/falcosecurity/charts/blob/master/falco/BREAKING-CHANGES.md#300)) comes with a new way to automatically update the Falco rules that are currently loaded. Of course, you can enable, disable and configure this functionality to your liking. Below, we list a number of common basic use cases and how to easily configure Falco for each:
```

**Review by jsalinas29 — COMMENTED** [review]
Just a few minor suggestions

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**